### DISCUSSION TITLE
problem with the installation on my Mac

### DISCUSSION THREAD

**Discussion by melanileo** [kaixxx/noScribe#54] [Q&A]
I have issues downloading the program on my MacBook. its says it cannot be open, since Mac can't search for bugges.I really need this program for my transcription.

**Comment by BabyFnord** [comment]
Please provide more information:
- maybe a screenshot of the error message
- macOS version
- Mac specs

**Comment by kaixxx** [comment]
Thank you, @BabyFnord, for stepping in! But the problem was already reslved here: https://github.com/kaixxx/noScribe/issues/55

I forgot to 'close' this discussion.
